XLS-0081d-permissioned-dex/README.md

```diff
@@ -0,0 +1,593 @@
+|`ledger_index`| |`string`|The ledger index of the ledger to use, or a shortcut string to choose a ledger automatically. The default is `validated`.|
+
+### 10.2. Response Fields
+
+As a reference, here are the fields that the [`book_changes` subscription currently returns](https://xrpl.org/docs/references/http-websocket-apis/public-api-methods/subscription-methods/subscribe#book-changes-stream):
+
+This proposal puts forward the following addition:
+
+| Field Name | Required? | JSON Type | Description |
+|------------|-----------|-----------|-------------|
+|`domain`| |`string`|The object ID of a `PermissionedDomain` object, if the orderbook changes are for a specific domain.|
+
+## 11. Examples
+
+These examples will be using the following domain:
+
+```typescript
+{
+  Owner: "rOWEN......",
+  Sequence: 5,
+  AcceptedCredentials: [
+    Credential: {
+      Issuer: "rISABEL......",
+      CredentialType: "123ABC"
+    }
+  ]
+}
+```
+
+The domain ID will be `ABCDEF1234567890`.
+
+Both Tracy and Marko have a `123ABC` credential from Isabel.
+
+### 11.1. Placing a Permissioned Offer
+
+In this sample `OfferCreate` transaction, Tracy is trading USD for EUR within Owen's domain.
+
+```typescript
+{
+  TransactionType: "OfferCreate",
+  Account: "rTRACY......",
+  TakerGets: {
+    currency: "USD",
+    issuer: "rUSDISSUER.......",
+    value: "11"
+  },
+  TakerPays: {
+    currency: "EUR",
+    issuer: "rEURISSUER......."
+    value: "10"
+  },
+  DomainID: "ABCDEF1234567890",
+  Fee: "12",
+  Sequence: 8
+}
+```
+
+### 11.2. Placing an Unpermissioned Offer
+
+In this sample `OfferCreate` transaction, Marko is trading EUR for USD on the open USD-EUR orderbook. This offer **will not cross** Tracy's offer, since Tracy's offer is inside the domain.
+
+```typescript
+{
+  TransactionType: "OfferCreate",
+  Account: "rMARKO......",
+  TakerGets: {
+    currency: "EUR",
+    issuer: "rEURISSUER.......",
+    value: "10"
+  },
+  TakerPays: {
+    currency: "USD",
+    issuer: "rUSDISSUER......."
+    value: "11"
+  },
+  Fee: "12",
+  Sequence: 3
+}
+```
+
+## 12. Invariants
+
+No permissioned offer with a `DomainID` field will be filled by:
+* An invalid domain offer.
+* An offer from a different domain.
+* An open offer.
+
+No permissioned offer will be placed in:
+* An orderbook for a different domain.
+* The open orderbook.
+
+No open offer will be placed in a permissioned orderbook.
+
+No open offer will be filled by a permissioned offer.
+
+## 13. Security
+
+The trust assumptions are the same as with permissioned domains.
+
+## n+1. Open Questions
+
+* Does a permissioned offer need to be able to be part of multiple domains? If so, the `DomainID` field should be replaced with a `DomainIDs` field (to accommodate a list)
+
+# Appendix
+
+## Appendix A: FAQ
+
+### A.1: How are AMMs handled?
+
+AMMs are not explicitly supported within permissioned DEXes in this proposal. They could be added to permissioned DEXes in a future proposal.
```

Comment:
does it mean that Payment transactions that specified DomainID cannot consume AMM pools?

Reply:
Correct. That's intentional.